frontend/src/layout/navigation-3000/components/AlgoliaSearch.tsx

```diff
@@ -0,0 +1,261 @@
+            </AutoSizer>
+        </ol>
+    )
+}
+
+const SearchInput = ({
+    value,
+    setValue,
+}: {
+    value: string
+    setValue: React.Dispatch<React.SetStateAction<string>>
+}): JSX.Element => {
+    const { refine } = useSearchBox()
+
+    const handleChange = (value: string): void => {
+        setValue(value)
+        refine(value)
+    }
+
+    return <LemonInput onChange={handleChange} value={value} type="search" fullWidth placeholder="Search..." />
+}
+
+type Tag = {
+    type: string
+    label: string
+}
+
+const tags: Tag[] = [
+    {
+        type: 'all',
+        label: 'All',
+    },
+    {
+        type: 'docs',
+        label: 'Docs',
+    },
+    {
+        type: 'question',
+        label: 'Questions',
+    },
+    {
+        type: 'tutorial',
+        label: 'Tutorials',
+    },
+]
+
+type SearchTagProps = Tag & {
+    active?: boolean
+    onClick: (type: string) => void
+}
+
+const SearchTag = ({ type, label, active, onClick }: SearchTagProps): JSX.Element => {
+    const { refine, items } = useRefinementList({ attribute: 'type' })
+    const itemCount = type !== 'all' && items.find(({ value }) => value === type)?.count
+
+    const handleClick = (e: React.MouseEvent<HTMLButtonElement, MouseEvent>): void => {
+        e.stopPropagation()
+        onClick(type)
+    }
+
+    useEffect(() => {
+        refine(type)
+    }, [])
+
+    return (
+        <button className="p-0 cursor-pointer bg-bg-light" onClick={handleClick}>
+            <LemonTag size="medium" type={active ? 'primary' : 'option'}>
+                <span>{label}</span>
+                {type !== 'all' && <span>({itemCount ?? 0})</span>}
+            </LemonTag>
+        </button>
+    )
+}
+
+const Tags = ({
+    activeTag,
+    setActiveTag,
+}: {
+    activeTag: string
+    setActiveTag: React.Dispatch<React.SetStateAction<string>>
+}): JSX.Element => {
+    const handleClick = (type: string): void => {
+        setActiveTag(type)
+    }
+
+    return (
+        <ul className="list-none m-0 p-0 flex space-x-1 mt-1 mb-0.5 pb-1.5 border-b px-2">
+            {tags.map((tag) => {
+                const { type } = tag
+                return (
+                    <li key={type}>
+                        <SearchTag {...tag} active={activeTag === type} onClick={handleClick} />
+                    </li>
+                )
+            })}
+        </ul>
+    )
+}
+
+const Search = (): JSX.Element => {
+    const { openSidePanel } = useActions(sidePanelStateLogic)
+    const { hits } = useHits()
+    const { items, refine } = useRefinementList({ attribute: 'type' })
+
+    const ref = useRef<HTMLDivElement>(null)
+    const [searchValue, setSearchValue] = useState<string>('')
+    const [activeOption, setActiveOption] = useState<undefined | number>()
+    const [activeTag, setActiveTag] = useState('all')
+    const [searchOpen, setSearchOpen] = useState(false)
+
+    const handleKeyDown = (e: React.KeyboardEvent<HTMLInputElement>): void => {
```

Comment:
Nit: Would be really intuitive if ArrowLeft/Right switched between All/Docs/Questions/Tutorials

Reply:
good idea, that sparks joy